lib/localize_ruby_client.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require "httparty"
+
+require_relative "client/configuration_helper"
+require_relative "client/version"
+require_relative "client/jwt_helper"
+require_relative "client/archive_helper"
+
+if defined?(Rails)
+  require "rails/railtie"
+  require_relative "client/railtie"
+end
+
+# The LocalizeRubyClient class serves as the main class of the LocalizeRubyClient gem.
+# It contains the core logic for interacting with the Localize API
+# This class utilizes HTTParty to perform HTTP requests to the Localize API,
+# providing methods to update translations, request translations, and upload files.
+class LocalizeRubyClient
+  include HTTParty
+  include Client::JwtHelper
+  include Client::ArchiveHelper
```

Comment:
This method can be added directly in the class I do not see the value of having them out, especially if they are only used here, and the class is less than 100 lines